### PR TITLE
[RFC] Fedora 30: Install gobject via pip for no system-wide packages

### PR DESCRIPTION
**Pull request by mikedep333**
See email thread on pulp-dev "Dependency on gi / GObject"

### CODE REVIEW

**Review by fao89 — APPROVED** [review]

**Comment by fao89** [comment]
this seems to solve this issue:
https://pulp.plan.io/issues/5992

**Review by dralley — APPROVED** [review]

**Comment by dralley** [comment]
Getting rid of all of the build failures in provisioning would be enormously helpful, thank you

File: vars/Fedora.yml [hunk 1]
```diff
@@ -5,18 +5,22 @@ packages:
...
   - python3-libmodulemd
   - rpm-devel
   - openssl-devel
   - sqlite-devel
   - xz-devel
   - zchunk-devel
   - zlib-devel
+
+rpm_prereq_pip_packages:
+  - PyGObject
```
Comment (dralley) [inline vars/Fedora.yml:26]: This is probably unnecessary as long as it's also listed in pulp_rpm's ```setup.py```

**Review by dralley — APPROVED** [review]

**Comment by mikedep333** [comment]
I recall that I decided against this because it made the install process larger & longer.

https://github.com/pulp/pulp_rpm_prerequisites/pull/14 is the preferred implementation.

**Comment by dralley** [comment]
@mikedep333  I don't think that patch fully fixes the issue, because I still can't vagrant up properly with the RPM plugin selected for installation.

**Comment by mikedep333** [comment]
@dralley Do you have `pulp_use_system_wide_pkgs` set to `true`?

I'm going to submit a PR for pulplift right now to update to the latest ansible-pulp also.

**Comment by dralley** [comment]
I checked and I do not.  Is it possible to set that variable as true automatically in the prerequisites plugin?

**Comment by mikedep333** [comment]
@dralley I want to look how to do that properly and robustly (handles the different ways users are calling the roles, like static roles list vs dynamic includes_roles tasks.)

We'll soon have CI on ansible-pulp for dynamic include_roles. That will help us made a decision.

**Pull request closed without merging**